### ISSUE TITLE
How to update the image in the imageview when the image is updated with same url in server.

### ISSUE THREAD

**Issue by ajay2c9** [pinterest/PINRemoteImage#437]
Hi,
I am loading a image in the imageview using imageurl. I updated the image in the same ImageUrl. how to update image in the imageview.

**Comment by garrettmoon** [comment]
@ajay2c9 I'm not sure I understand the question, can you rephrase it? Have you looked at the example project: https://github.com/pinterest/PINRemoteImage/tree/master/Examples/Example?
